www/include/common/javascript/xslt.js

```diff
@@ -101,7 +101,7 @@ function loadXML(url) {
         xmlDoc.async = false;
         xmlDoc.load(url);
       } else if (window.XMLHttpRequest) {
-		// ï¿½ l'aide de lobjet XMLHTTPRequest
+		// à l'aide de l'objet XMLHTTPRequest
```

Comment:
I think this comment can be definitely removed :D 
Cause it does not help at all
```suggestion
```